sublime_jedi/tooltips/markdown.py

```diff
@@ -53,17 +53,20 @@ def _prepare_signature(self, docstring):
         """
         pattern = (
             '(?x)'
-            '^([\w\. \t]+\.[ \t]*)?'        # path
-            '(\w+)'                         # function / object
-            '[ \t]*(\([^\)]*\))'            # arguments
-            '(\s*->\s*.*$)?'                # annotation (rest of line)
+            '^([\w\. \t]+\.[ \t]*)?'    # path
+            '(\w+)'                     # function / object
+            '[ \t]*(\([^\)]*\))'        # arguments
+            '(?:'
+            '(\s*->\s*.*?)'             # annotation: -> Type
+            '(--|$)'                    # inline comment: -- blabla
+            ')?'
```

Comment:
> but should we fix (drop) any part of docstring, if the part is formatted incorrectly?

Do you mean the inline stuff after `--` ?

I just move it to the first line of the docstring at the moment. Don't drop anything.

> why you chose mdpopups?

It does the code highlighting of the signature.

The screenshot shows the highlighted L.extend() signature with its "inline comment" being moved to the next line.

![screenshot](https://user-images.githubusercontent.com/16542113/26877731-23ff07f2-4b8b-11e7-9fb5-78f3f055453a.png)